cat-launcher/src-tauri/src/utils.rs

```diff
@@ -117,15 +117,63 @@ pub fn manage_repositories(app: &App) -> Result<(), RepositoryError> {
   Ok(())
 }
 
+async fn migrate_backup_directory(
```

Comment:
Create a general, migrate_to_local_dir function which should copy all files recursively into the local app dir from the app dir, taking care not to overwrite anything.